code/modules/vore/eating/belly_obj_vr.dm

```diff
@@ -41,6 +41,7 @@
 	var/escape_stun = 0						// AI controlled mobs with a number here will be weakened by the provided var when someone escapes, to prevent endless nom loops
 	var/transferchance = 0 					// % Chance of prey being trasnsfered, goes from 0-100%
 	var/transferchance_secondary = 0 		// % Chance of prey being transfered to transferchance_secondary, also goes 0-100%
+	var/belchchance = 0						//CHOMPAdd % Chance of pred belching on prey struggle
```

Comment:
personally I would prefer that in belly_obj_ch, the rest looks good, now where I have the time to go through it properly

Reply:
Good to hear, I can see why that would be better placement. I'll push that change through in a moment.